spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/TestSparkDistributionAndOrderingUtil.java

```diff
@@ -1386,14 +1801,27 @@ public void testRangeCopyOnWriteMergePartitionedSortedTable() {
   // Distribution and ordering for merge-on-read DELETE operations with position deletes
   // ===================================================================================
   //
-  // delete mode is NOT SET -> CLUSTER BY _spec_id, _partition + LOCALLY ORDER BY _spec_id,
-  // _partition, _file, _pos
-  // delete mode is NONE -> unspecified distribution + LOCALLY ORDER BY _spec_id, _partition, _file,
-  // _pos
-  // delete mode is HASH -> CLUSTER BY _spec_id, _partition + LOCALLY ORDER BY _spec_id, _partition,
-  // _file, _pos
+  // UNPARTITIONED (ORDERED & UNORDERED)
+  // -------------------------------------------------------------------------
+  // delete mode is NOT SET -> CLUSTER BY _spec_id, _partition, _file +
```

Comment:
Adding `_file` to the clustering for unpartitioned tables as `spec_id`, `_partition` will be the same for all rows and deletes for the same data file may end up in multiple delete files.